### PR TITLE
Match multiline function declarations to devbook

### PR DESCRIPTION
**Pull request by arrrnas**
since https://github.com/vinted/devbook/pull/201 was agreed in 2016 let's discuss if this makes sense in 2019 @vinted/ios

### CODE REVIEW

**Review by ricardozauras — APPROVED** [review]

**Review by NerijusLas — APPROVED** [review]

**Review by Gurbonas — APPROVED** [review]

File: README.markdown [hunk 1]
```diff
@@ -479,11 +479,10 @@ func reticulateSplines(spline: [Double]) -> Bool {
 For functions with long signatures, put each parameter on a new line and add an extra indent on subsequent lines:
 
 ```swift
-func reticulateSplines(
-  spline: [Double], 
-  adjustmentFactor: Double,
-  translateConstant: Int, comment: String
-) -> Bool {
+func reticulateSplines(spline: [Double],
```
Comment (daumantas-versockas-vinted) [inline README.markdown:482]: I like first one better. IMHO, there is a difference between declarations and calls (`func`, parameters, return type). So, I like them to look similar (the same function...)

Comment (justas-vinted) [inline README.markdown:482]: when should we move parameters into separate lines? What is "long signature"?

**Review by justas-vinted — APPROVED** [review]

File: README.markdown (hunk 1 above)
Comment (arrrnas) [reply README.markdown:482]: we can use https://github.com/realm/SwiftLint/blob/master/Rules.md#multiline-parameters to enforce this, either all parameters on the same line ar all in separate lines with the first parameter being on the same line as `func`

**Comment by arrrnas** [comment]
merging based on majority vote

**Pull request merged**